docs/EventData.md

```diff
@@ -79,6 +79,18 @@ The event object has the following properties:
   };
 ```
 
+## SubscriberCaptionEvent
+
+To get captions for a subscriber, register an event listener for the `captionReceived` event.
+The event object has the following properties: 
+
+```javascript
+  event = {
+    text: string,
```

Comment:
For JS SDK, the caption is returned as `caption` but for React Native, it's returned as `text`.  Was this intentional?